src/features/main/Start.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+
+import startIcon from "../../assets/icon-start.svg";
+
+const StyledStart = styled.div`
+  width: 100%;
+  height: 100%;
+  min-height: 568px;
+  background-color: #0000008e;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  .wrapper {
```

Comment:
css 를 styled 로 선언할때 저는 클래스로 선언한 부분은 항상 개행을 해주는 편인데요 이부분은 합의가 필요해 보입니다. 
```suggestion

  .wrapper {
```

Reply:
저도 같은 생각 입니다..

Reply:
저도 개행을 해주는 스타일이었는데 너무 길어져서 ^^;; 이야기 해보시죠!

Reply:
일단 이번 커밋은 수정하는대로 바로 진행할게요!